### ISSUE TITLE
Error After Payment Completion 

### ISSUE THREAD

**Issue by seyi-adetunji** [PaystackHQ/Paystack-gateway-for-Prestashop-1.7#7]
Hi Support,
                 I currently installed PrestaShop 7.1 paystack integration, After the customer completes the payment process and the account debited, there is usually a redirect error from paystack back to the website that gives payment successful feedback. Please, can you confirm what we need to do at our end as well as yours  too.

**Comment by steveamaza** [comment]
Hello @seyi-adetunji, we will need some more information. What specific redirect error is happening? Can you provide screenshots? Entry from your server error logs?

(PS: I think you mean PrestaShop 1.7. Please correct me if I am wrong so I can look at the issue with fresh eyes)

**Comment by seyi-adetunji** [comment]
Yes. Prestashop 1.7. When a customer completes a successful payment and instead of getting order payment confirmation, they get server error page.

**Comment by marvelefe** [comment]
Please this error still hasn't been fixed. Prestashop 1.7.

**Comment by RooseveltTech** [comment]
I have been able to fix this. I would like to update the paystack code for pretashop 8.1
